### ISSUE TITLE
PDF generated by Story doesn't contain image on iOS/Android

### ISSUE THREAD

**Issue by llaske** [llaske/sugarizer#969]
When a PDF is generated by Story activity on iOS/Android, the PDF does not contain image (image are blanks).

![Peek 12-01-2022 22-19](https://user-images.githubusercontent.com/981841/149224695-21db8235-2a3c-4e33-b935-54d53ba8caa1.gif)

It doesn't happen on the web version.
It doesn't happen when the package is not minimized i.e. when all Abecedarium images are include in the package (using [minsize](https://github.com/llaske/sugarizer-apkbuilder#generate-your-apk) option in APK builder).

This latest point suggest me that it could be related to the issue described in the comment [here](https://github.com/llaske/sugarizer/pull/960#issuecomment-939356420).

**Comment by llaske** [comment]
@saurabhhere any idea?

**Comment by saurabhhere** [comment]
Hi @llaske , but that issue in handled in this [commit](https://github.com/llaske/sugarizer/pull/960/commits/f6c72cce7c6b2b99137aae837401b73b17caa061), and after that I tested it even when the package is minimized and it worked fine.

**Comment by llaske** [comment]
@saurabhhere yes the visualization of images work (as you could see in the video), the issue is now on PDF generation only.

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/commit/9e98396fd6daba05b25bf20c794bd9731e9d6271
